tests/distributed/test_new_kvstore.py

```diff
@@ -0,0 +1,179 @@
+import os
+import time
+import numpy as np
+from scipy import sparse as spsp
+import dgl
+import backend as F
+import unittest, pytest
+from dgl.graph_index import create_graph_index
+
+from numpy.testing import assert_array_equal
+
+def create_random_graph(n):
+    arr = (spsp.random(n, n, density=0.001, format='coo') != 0).astype(np.int64)
+    ig = create_graph_index(arr, readonly=True)
+    return dgl.DGLGraph(ig)
+
+# Create an one-part Graph
+node_map = F.tensor([0,0,0,0,0,0], F.int64)
+edge_map = F.tensor([0,0,0,0,0,0,0], F.int64)
+global_nid = F.tensor([0,1,2,3,4,5], F.int64)
+global_eid = F.tensor([0,1,2,3,4,5,6], F.int64)
+
+g = dgl.DGLGraph()
+g.add_nodes(6)
+g.add_edge(0, 1) # 0
+g.add_edge(0, 2) # 1
+g.add_edge(0, 3) # 2
+g.add_edge(2, 3) # 3
+g.add_edge(1, 1) # 4
+g.add_edge(0, 4) # 5
+g.add_edge(2, 5) # 6
+
+g.ndata[dgl.NID] = global_nid
+g.edata[dgl.EID] = global_eid
+
+gpb = dgl.distributed.GraphPartitionBook(part_id=0,
+                                         num_parts=1,
```

Comment:
can you test the multi-partition case?

Reply:
Multi-partition need real distributed setting. I already tested it.